src/Compilers/CSharp/Test/Symbol/Symbols/DefaultInterfaceImplementationTests.cs

```diff
@@ -3919,5 +3919,1172 @@ class Test1 : I1
+    event System.Action E7 
+    { 
+        add {} 
+        remove {} 
+    }
+}
+
+class Test1 : I1
+{}
+";
+            var compilation1 = CreateCompilationWithMscorlib(source1, options: TestOptions.DebugDll,
+                                                 parseOptions: TestOptions.Regular.WithLanguageVersion(LanguageVersion.CSharp7));
+            Assert.True(compilation1.Assembly.RuntimeSupportsDefaultInterfaceImplementation);
+
+            compilation1.VerifyDiagnostics(
+                // (6,9): error CS8107: Feature 'default interface implementation' is not available in C# 7.  Please use language version 7.1 or greater.
+                //         add {} 
+                Diagnostic(ErrorCode.ERR_FeatureNotAvailableInVersion7, "add").WithArguments("default interface implementation", "7.1").WithLocation(6, 9),
+                // (7,9): error CS8107: Feature 'default interface implementation' is not available in C# 7.  Please use language version 7.1 or greater.
+                //         remove {} 
+                Diagnostic(ErrorCode.ERR_FeatureNotAvailableInVersion7, "remove").WithArguments("default interface implementation", "7.1").WithLocation(7, 9)
+                );
+
+            ValidateEventImplementation_501(compilation1.SourceModule, "Test1");
+
+            var source2 =
+@"
+class Test2 : I1
+{}
+";
+
+            var compilation2 = CreateCompilationWithMscorlib(source2, new[] { compilation1.ToMetadataReference() }, options: TestOptions.DebugDll,
+                                                            parseOptions: TestOptions.Regular.WithLanguageVersion(LanguageVersion.CSharp7));
+            Assert.True(compilation2.Assembly.RuntimeSupportsDefaultInterfaceImplementation);
+            compilation2.VerifyDiagnostics();
+
+            ValidateEventImplementation_501(compilation2.SourceModule, "Test2");
+
+            CompileAndVerify(compilation2, verify: false,
+                symbolValidator: (m) =>
+                {
+                    var test2Result = (PENamedTypeSymbol)m.GlobalNamespace.GetTypeMember("Test2");
+                    Assert.Equal("I1", test2Result.Interfaces.Single().ToTestDisplayString());
+                    ValidateEventImplementation_501(m, "Test2");
+                });
+        }
+
+        [Fact]
+        public void EventImplementation_901()
+        {
+            var source1 =
+@"
+public interface I1
+{
+    static event System.Action E7 
+    { 
+        add {} 
+        remove {} 
+    }
+}
+
+class Test1 : I1
+{}
+";
+            var compilation1 = CreateCompilationWithMscorlib(source1, options: TestOptions.DebugDll,
+                                                 parseOptions: TestOptions.Regular.WithLanguageVersion(LanguageVersion.CSharp7));
+            Assert.True(compilation1.Assembly.RuntimeSupportsDefaultInterfaceImplementation);
+
+            compilation1.VerifyDiagnostics(
+                // (4,32): error CS0106: The modifier 'static' is not valid for this item
```

Comment:
I know it was not your intent to support it yet, but the appropriate diagnostic here should be that `'static' in an interface is not supported in language version '7'; use language version '7.1'`. Please add a `PROTOTYPE` comment noting that will need to change.